output.md

```diff
@@ -90,59 +95,73 @@ file.h5
         ...
     snapshot_1/
         attribute: t
-        attribute: ts 
+        attribute: ts
         space_1/
         ...
         space_n/
     ...
     snapshot_n/
 ```
-In addition to that, psydac also has a class to read those files, recreate all the `FemSpace` and `FemField` objects and export them to `VTK`. 
+In addition to that, psydac also has a class to read those files, recreate all the `FemSpace` and `FemField` objects and export them to `VTK`.
 
 ## Usage of class `OutputManager`
 
 An instance of the `OutputManager` class is created at the beginning of the simulation, by specifying the following:
 
-1.  The name of the YAML file (e.g. `spaces.yml`) where the information about all FEM spaces will be written, and
+1.  The name of the YAML file (e.g. `spaces.yaml`) where the information about all FEM spaces will be written, and
 2.  The name of the HDF5 file (e.g. `fields.h5`) where the coefficients of all FEM fields will be written.
 
-References to the available FEM spaces are given to the OutputManager object through the `add_spaces(**kwargs)` method, and the corresponding YAML file is created upon calling the method `export_spaces_info()`. In order to inform the OutputManager object that the next fields to be exported are time-independent, the user should call the `set_static()` method. In the case of time-dependent fields, the user should prepare a time snapshot (which is defined for a specific time step index `ts` and time value `t`) by calling the method `add_snapshot(t, ts)`. In both cases the fields are exported to the HDF5 file through a call to the method `export_fields(**kwargs)`. Here is a usage example:
+References to the available FEM spaces are given to the OutputManager object through the `add_spaces(**kwargs)` method, and the corresponding YAML file is created upon calling the method `export_spaces_info()`. In order to inform the OutputManager object that the next fields to be exported are time-independent, the user should call the `set_static()` method. In the case of time-dependent fields, the user should prepare a time snapshot (which is defined for a specific time step `ts` and time value `t`) by calling the method `add_snapshot(t, ts)`. In both cases the fields are exported to the HDF5 file through a call to the method `export_fields(**kwargs)`. Here is a usage example:
```

Comment:
- Typo: export_spaces_info -> export_space_info (singular)
- If you do not mention that `ts` is an index, you should at least say that it is an integer

Reply:
Fixed in [34bfd98](https://github.com/pyccel/psydac/pull/234/commits/34bfd9845a7a7c44d8589ff9b670b919f204d457)